chain/chain/src/types.rs

```diff
@@ -172,7 +174,10 @@ impl BlockEconomicsConfig {
     pub fn max_gas_price(&self, protocol_version: ProtocolVersion) -> Balance {
         if checked_feature!("protocol_feature_cap_max_gas_price", CapMaxGasPrice, protocol_version)
         {
-            std::cmp::min(self.max_gas_price, 10 * self.min_gas_price(protocol_version))
+            std::cmp::min(
+                self.max_gas_price,
+                Self::MAX_GAS_MULTIPLIER * self.min_gas_price(protocol_version),
```

Comment:
Not likely to matter, but maybe do checked multiplication?

Reply:
well we know the constants here and they will not cause overflow